README.md

```diff
@@ -1,64 +1,88 @@
+Visit the [Downloads page](https://cesium.com/downloads/) to download a pre-built copy of CesiumJS.
 
-### :rocket: Get Started
+### npm & yarn
 
-Visit the [Downloads page](https://cesium.com/downloads/) or install the [npm package](https://www.npmjs.com/package/cesium):
+If you’re building your application using a module bundler such as Webpack, Parcel, or Rollup, you can install CesiumJS via the [`cesium` npm package](https://www.npmjs.com/package/cesium):
 
 ```sh
-npm install cesium
+npm install cesium --save
+```
+
+Then, import CesiumJS in your app code:
+
+```js
+import * as Cesium from "cesium";
+import "cesium/Build/Cesium/Widgets/widgets.css";
+
+const viewer = new CesiumWidgets.Viewer("cesiumContainer");
+```
+
+Or, import individual modules to benefit from tree shaking optimizations through most build tools:
+
+```js
+import { Viewer } from "cesium";
+import "cesium/Build/Cesium/Widgets/widgets.css";
+
+const viewer = new Viewer("cesiumContainer");
 ```
 
-Have questions? Ask them on the [community forum](https://community.cesium.com/).
+#### Packages
+
+In addition to the `cesium` package, CesiumJS is also distributed as scoped npm packages for more specific dependency management:
+
+- [`@cesium/engine`](./packages/engine/README.md) - CesiumJS's core, rendering, and data APIs
+- [`@cesium/widgets`](./packages/widgets/README.md) - A widgets library for use with CesiumJS
+
+### CDN
+
+Alternatively, [use a content delivery network (CDN) URL](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/#import-from-cdn) to embed CesiumJS in an HTML page.
+
+### What next?
+
+See our [Quickstart Guide](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/) for more information on getting a CesiumJS app up and running.
+
+Instructions for serving local data are in the CesiumJS
+[Offline Guide](./Documentation/OfflineGuide/README.md).
 
 Interested in contributing? See [CONTRIBUTING.md](CONTRIBUTING.md). :heart:
 
-### :snowflake: Mission
+## :snowflake: Mission
 
 Our mission is to create the leading 3D globe and map for static and time-dynamic content, with the best possible performance, precision, visual quality, platform support, community, and ease of use.
 
-### :green_book: License
+## :green_book: License
 
 [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0.html). CesiumJS is free for both commercial and non-commercial use.
 
-### :earth_americas: Where Does the 3D Content Come From?
-
-CesiumJS can stream 3D content such as terrain, imagery, and 3D Tiles from the commercial [Cesium ion](https://cesium.com/blog/2018/03/01/hello-cesium-ion/)
-platform and other content sources. You are free to use any combination of content sources with CesiumJS that you please.
-Using Cesium ion helps support CesiumJS development. :heart:
-
-### :clap: Featured Demos
-
-<p>
-<a href="https://cesium.com/blog/2018/08/21/cybercity/"><img src="https://images.prismic.io/cesium/2018-08-21-cover.jpg" width="30%" /></a>&nbsp;
```

Comment:
We should link to these somewhere if applicable. Maybe also supplement them with linking to User Stories.

Reply:
Would a single link be sufficient? I'm thinking of adding a "User Stories" link to the top group of links further up the page. This is similar to [how Three links their examples](https://github.com/mrdoob/three.js/blob/dev/README.md).